presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1243,17 +1243,58 @@ public void testJoinUsing()
     }
 
     @Test
+    public void testJoinUsingOutputColumn()
+            throws Exception
+    {
+        assertQuery(
+                "SELECT * FROM (VALUES (1)) a (x) JOIN (VALUES (1)) b (x) USING (x)",
+                "VALUES (1)");
+
+        assertQuery(
+                "SELECT x FROM (VALUES (1)) a (x) JOIN (VALUES (1)) b (x) USING (x)",
+                "VALUES (1)");
+
+        assertQuery(
+                "SELECT x FROM (VALUES (1), (2)) a (x) FULL JOIN (VALUES (2), (3)) b (x) USING (x)",
+                "VALUES (1, 2, 3)");
```

Comment:
This test is wrong. Should be `VALUES 1, 2, 3` (which is equivalent `VALUES (1), (2), (3)`). Additionally, this test fails before and after fixing this line.